src/ngraph_mark_for_clustering.cc

```diff
@@ -302,6 +302,7 @@ Status MarkForClustering(Graph* graph) {
       type_constraint_map["Transpose"]["T"] = NGraphDTypes();
       type_constraint_map["Transpose"]["Tperm"] = NGraphIndexDTypes();
       type_constraint_map["Unpack"]["T"] = NGraphDTypes();
+      type_constraint_map["ZerosLike"]["T"] = NGraphNumericDTypes();
 
```

Comment:
The registration of zeros like: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/array_ops.cc#L736

It accepts all types of data even booleans. Hence we should not use NGraphNumericDTypes();
We can skip this line totally, as there is no type constraint.